lib/bokken/uploaders/document.ex

```diff
@@ -4,8 +4,18 @@ defmodule Bokken.Uploaders.Document do
   """
   use Waffle.Definition
   use Waffle.Ecto.Definition
+  alias Bokken.Documents.File
+  @extension_whitelist ~w(.jpg .jpeg .gif .png)
+  @max_file_size 5_000_000
```

Comment:
```suggestion
  @max_file_size 4_000_000
```

If the maximum is 6MB, it doesn't make much sense to put 2 for an avatar and 5 for a document (2 + 5 = 7 > 6).